storage/gluster-deploy.sh

```diff
@@ -0,0 +1,40 @@
+#!/bin/bash
+#
+# This file is part of the KubeVirt project
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+# Copyright 2017 Red Hat, Inc.
+#
+
+
+PROVIDER=${PROVIDER:-vagrant-kubernetes}
```

Comment:
All that looks like it can be done from an init container, or directly by the cluster container in the manifest.

Reply:
ok now I get it. gk-deploy does some deployment in k8s. That all looks way too complicated.

Reply:
pulling in a 1000 line bash script is concerning.

Reply:
It's what we have now. It's been verified and used in enterprise production environments. We're looking ahead to changing this, but it's not something we'll get to soon.